Neos.ContentRepository/Classes/Security/Authorization/Privilege/Node/Doctrine/ConditionGenerator.php

```diff
@@ -86,6 +87,17 @@ public function isDescendantNodeOf($nodePathOrIdentifier)
         return new DisjunctionGenerator([$propertyConditionGenerator1->like($nodePath . '/%'), $propertyConditionGenerator2->equals($nodePath)]);
     }
 
+    /**
+     * @param array $nodeTypes
+     * @return PropertyConditionGenerator
+     */
+    public function isDescendantOfNodetype($nodeTypes)
+    {
+        $propertyConditionGenerator1 = new DecendantOfNodetypeConditionGenerator($nodeTypes);
+       
```

Comment:
```
return new DescentantOfNodeTypeConditionGenerator(...);
```

=> and I'd suggest to put the DescendantOfNodeTypeConditionGenerator() into Neos.ContentRepository/Classes/Security/Authorization/Privilege/Node/Doctrine/... -> no need to place this in Flow IMHO (and it would make Flow depend on Neos, which we want the other way around.